squirtgun-api/src/main/java/me/lucyy/squirtgun/platform/scheduler/Task.java

```diff
@@ -128,4 +147,40 @@ public void execute(Platform platform) {
 	public boolean isAsync() {
 		return isAsync;
 	}
+
+	public long getId() {
+		return this.id;
+	}
+
+	@Override
+	public String toString() {
+		return "Task{"
+					 + "id=" + this.id
+					 + ", action=" + this.action
+					 + ", delay=" + this.delay
+					 + ", interval=" + this.interval
+					 + ", isAsync=" + this.isAsync
+					 + '}';
+	}
+
+	@Override
+	public boolean equals(final Object other) {
+		if (this == other) { return true; }
+		if (!(other.getClass() == this.getClass())) { return false; }
+
+		final Task that = (Task) other;
+		return this.delay == that.delay
+					 && this.interval == that.interval
+					 && this.isAsync == that.isAsync
+					 && this.id == that.id;
+	}
+
+	@Override
+	public int hashCode() {
+		int result = this.delay;
```

Comment:
would Objects.hash not be easier?

Reply:
Maybe, it's auto-generated anyway and "[it just works](https://stackoverflow.com/a/1646913/8083725)", can be changed if requested

Reply:
Went ahead and changed it https://github.com/lucyy-mc/Squirtgun/pull/29/commits/87538d8c7838cbc7ddf8c27639c688dd2a6a6149